app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>PaasAdmin</title>
+    <%= csrf_meta_tags %>
+
+    <%= stylesheet_link_tag    'application', media: 'all' %>
+    <%= javascript_include_tag 'application' %>
+  </head>
+
+  <body>
+    <img src="https://static1.squarespace.com/static/59698a7d29687fd47a2a7c52/599a2c9837c581a0f21a7030/5998fbff17bffc5c4b7d0f04/1503700549384/construction_worker.jpg" border="0" width="125">
```

Comment:
Should we vendor this rather than hotlink?

Reply:
:+1: We should be hosting that ourselves - makes us better Internet citizens, and also we can't be sure of the longevity of that link etc.

Reply:
I love your optimism that Construction Duck will last persist long enough to be worried about this.